context-extraction/src/main/java/de/atb/context/extraction/ContextContainer.java

```diff
@@ -148,7 +143,7 @@ public final void addDefaultModel(BusinessCase bc) {
         } else {
             model.read(OntologyNamespace.getInstance().getAbsoluteUri());
         }
-        add(model, true);
+        // TODO add(model, true);
```

Comment:
is something missing here?

Reply:
the method signature changed. Removed the second parameter to be compliant with new signature.